jekyll/_includes/js-assets.html

```diff
@@ -1,13 +1,13 @@
 <div>
-  {% asset vendor.min.js %}
-  <script src="https://unpkg.com/@popperjs/core@2.4.4/dist/umd/popper.min.js"></script>
+  <script type="text/javascript" src="{{ site.baseurl }}/{% ministamp assets/js/vendor.min.js assets/vendor.min.js %}"></script>
+  <script type="text/javascript" src="https://unpkg.com/@popperjs/core@2.4.4/dist/umd/popper.min.js"></script>
   {% include_cached rollbar.html site=site %}
-  {% asset app.bundle.js %}
-  {% asset site.min.js %}
-  {% asset jquery.cookie.min.js %}
-  {% asset main.js %}
-  {% asset nav.js %}
-  {% asset shared/components/subnav.js %}
-  {% asset shared/analytics-recursive-tracking.js %}
+  <script type="text/javascript" src="{{ site.baseurl }}/{% ministamp assets/js/app.bundle.js assets/app.bundle.js %}"></script>
+  <script type="text/javascript" src="{{ site.baseurl }}/{% ministamp assets/js/site.min.js assets/site.min.js %}"></script>
+  <script type="text/javascript" src="{{ site.baseurl }}/{% ministamp assets/js/jquery.cookie.min.js assets/jquery.cookie.min.js %}"></script>
+  <script type="text/javascript" src="{{ site.baseurl }}/{% ministamp assets/js/main.js assets/main.js %}"></script>
+  <script type="text/javascript" src="{{ site.baseurl }}/{% ministamp assets/js/nav.js assets/nav.js %}"></script>
```

Comment:
We'll need to add:
`  <script type="text/javascript" src="{{ site.baseurl }}/{% ministamp assets/js/sidebar.js assets/sidebar.js %}"></script>`

As `main.js` uses a `Sprockets` require (`//=  require sidebar.js`) (which we'll also want to remove)

Reply:
Oh, right... Makes sense